sdcm/cluster.py

```diff
@@ -5850,7 +5850,7 @@ def stop_scylla_monitoring(self, node):
         """)
         node.remoter.run("bash -ce '%s'" % kill_script)
 
-    def get_grafana_screenshot_and_snapshot(self, test_start_time: Optional[int] = None) -> dict[str, list[str]]:
+    def get_grafana_screenshot_and_snapshot(self, test_start_time: Optional[float] = None) -> dict[str, list[str]]:
```

Comment:
so `snapshot` in this name also is misleading. Worth to consider renaming this method.

Reply:
changed the name